class/practice_10.1/include/team.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include <string>
+
+class Team {
+public:
+    std::string name;
+    int goals_scored = 0;
+    int goals_conceded = 0;
+    int yellow_cards = 0;
+    int red_cards = 0;
+    double top_heights = 0.0;
+    double points = 0.0;
+
+    Team(const std::string& name);
```

Comment:
explicit for constructor with a single parameter maybe helpful to avoid implicit conversions like:
```
const std::string someString = ok;
Team myTeam("Ukraine");
myTeam = someString; // will call constructor implicitly 
```